xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
-    client, the following environment variables must be set on the client side:
-   </para>
-<screen>CVS_RSH=ssh CVSROOT=&exampleuser_plain;@server:/serverdir</screen>
-   <para>
-    The command <command>cvs</command>&nbsp;<option>init</option> can be used
-    to initialize the CVS server from the client side. This needs to be done
-    only once.
-   </para>
-   <para>
-    Finally, the synchronization must be assigned a name. Select or create a
-    directory on the client to contain files to manage with CVS (the directory
-    can also be empty). The name of the directory is also the name of the
-    synchronization. In this example, the directory is called
-    <literal>synchome</literal>. Change to this directory and enter the
-    following command to set the synchronization name to
-    <literal>synchome</literal>:
-   </para>
-<screen>cvs import synchome &exampleuser_plain; &exampleuserII_plain;</screen>
-   <para>
-    Many CVS commands require a comment. For this purpose, CVS starts an editor
-    (the editor defined in the environment variable <literal>$EDITOR</literal>
-    or vi if no editor was defined). The editor call can be circumvented by
-    entering the comment in advance on the command line, such as in the
-    following example:
-   </para>
-<screen>cvs import -m 'this is a test' synchome &exampleuser_plain; &exampleuserII_plain;</screen>
-  </sect2>
+  <!-- Use cases and properties of Rsync? -->
 
-  <sect2 xml:id="sec.net.sync.cvs.using">
-   <title>Using CVS</title>
-   <para>
-    The synchronization repository can now be checked out from all hosts with
-    <command>cvs</command>&nbsp;<option>co synchome</option>. This creates a
-    new subdirectory <filename>synchome</filename> on the client. To commit
-    your changes to the server, change to the directory
-    <filename>synchome</filename> (or one of its subdirectories) and enter
-    <command>cvs</command>&nbsp;<option>commit</option>.
-   </para>
+  <sect2 xml:id="sec.net.sync.rsync.syntax">
+   <title>Basic Syntax</title>
    <para>
-    By default, all files (including subdirectories) are committed to the
-    server. To commit only individual files or directories, specify them as in
-    <command>cvs</command>&nbsp;<option>commit file1 directory1</option>. New
-    files and directories must be added to the repository with a command like
-    <command>cvs</command>&nbsp;<option>add file1 directory1</option> before
-    they are committed to the server. Subsequently, commit the newly added
-    files and directories with <command>cvs</command>&nbsp;<option>commit file1
-    directory1</option>.
+    Rsync is a command line tool which the following basic syntax:
    </para>
+   <screen>rsync [OPTION] SOURCE [SOURCE]... DEST</screen>
+   <para> You can use Rsync to any machine that you have access to and write
+    permissions, be it local or remote. <replaceable>SOURCE</replaceable> can
```

Comment:
<replaceable>SOURCE</replaceable> can be added one or many times.

**It is possible to have multiple <replaceable>SOURCE</replaceable>  entries.